note/checkstyle-sample-note.md

```diff
@@ -222,7 +222,9 @@ Check that it's working by writing some poorly styled code:
 
     FAILURE: Build failed with an exception.
 
-### Step 5 - Check before launch
+### Step 5 - Check before launch (optional)
+
+(After experience, I found this setting affects the develop efficiency too much, so make the step optional.)
```

Comment:
In fact, I think should delete this step. after I enable it, when develop, it will do the check every time I want to run the app for testing, and you know the gradle is very slow, especially in China.